disperser/apiserver/server.go

```diff
@@ -605,7 +603,12 @@ func (s *DispersalServer) updateQuorumConfig(ctx context.Context) (QuorumConfig,
 			return QuorumConfig{}, err
 		}
 	} else {
-		newConfig.SecurityParams = securityParams
+		if newConfig.SecurityParams == nil {
```

Comment:
Why is this part being changed? The original approach here was meant to avoid race conditions by replacing the map instead of writing to the same map. 

The new approach seems like it introduces a risk of a race condition, and I'm unsure what the intended purpose is.

Reply:
* Previously it was a list, but the same point applies.

Reply:
I guess the purpose was probably to convert this to a map from a list. I'm not sure the map is needed, since the security param array is dense. If we're going with the map, I'd suggest just creating a new map every time?

Reply:
Thanks for catching this. Patch in https://github.com/Layr-Labs/eigenda/pull/366